lte/gateway/deploy/agwc-helm-charts/README.md

```diff
@@ -16,6 +16,32 @@ The following table list the configurable parameters of the agw chart and their
 | `image.tag` | Tag for agwc image. | `latest` |
 | `config.domainName` | Orchestrator domain name. | `` |
 | `persistant.name` | Secret name containing agwc certs. | `agwc-claim` |
+| `config.gwChallenge` | Challenge key. | `` |
```

Comment:
The `gwChallenge` will get created automatically, or we have to create is manually first?

Reply:
I see in the docs you have mentioned that we have to create `rootCA.pem`, `snowflake` and `gw_challenge.pem` manually first. Is that right?

Reply:
Can't we generate this automatically using a Kubernetes job and then deploy the helm charts?

Reply:
we've to manually create `gw_challenge.pem` and `snowflake` first.
`rootCA.pem` from orc8r deployment.
and sure, we can have a K8s job to automate this.
will add a job soon.
Thanks

Reply:
Thanks @Alt-Shivam